dts/bindings/dma/st,stm32-dma-v2.yaml

```diff
@@ -42,6 +42,12 @@ description: |
                0x2: high
                0x3: very high
     4. Features. Not used. Kept for binding compatibility with "st,stm32-dma-v1"
+        - bit 3: Source Reload
```

Comment:
Could this config be set at bit position 5 of the channel-config, just matching the  **CIRC** bit definition of the DMA channel x configuration register -->   the reload-en for source and dest and 
instead of extending the "feature" which was reserved for the fifo control on onther version of DMA instance.